docs/reference/supported-environments.md

```diff
@@ -50,35 +68,58 @@ Requirements for the components can be seen below:
 
+<!-- Nit pick - this is a list, not a matrix. This is also on a separate site with no oversight from DevEx. -->
+
 For more details about the applications version included in the Helm chart, check out the [full version matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
 
 ### Applications version matrix
 
-This overview shows which Zeebe version works with which Modeler, Operate, Tasklist and Optimize:
-
-| Design                | Automate    |                                             | Improve         | form-js |
-| --------------------- | ----------- | ------------------------------------------- | --------------- | ------- |
-| Desktop Modeler 4.7+  | Zeebe 1.0.x | Operate 1.0.x Tasklist 1.0.x                | -               | 0.0.1   |
-| Desktop Modeler 4.9+  | Zeebe 1.1.x | Operate 1.1.x Tasklist 1.1.x                | -               | 0.1.x   |
-| Desktop Modeler 4.11+ | Zeebe 1.2.x | Operate 1.2.x Tasklist 1.2.x IAM 1.2.x      | -               | 0.1.x   |
-| Desktop Modeler 4.12+ | Zeebe 1.3.x | Operate 1.3.x Tasklist 1.3.x IAM 1.3.x      | Optimize 3.7.x  | 0.1.x   |
-| Desktop Modeler 5.0+  | Zeebe 8.0.x | Operate 8.0.x Tasklist 8.0.x Identity 8.0.x | Optimize 3.8.x  | 0.2.x   |
-| Desktop Modeler 5.4+  | Zeebe 8.1.x | Operate 8.1.x Tasklist 8.1.x Identity 8.1.x | Optimize 3.9.x  | 0.8.x   |
-| Desktop Modeler 5.10+ | Zeebe 8.2.x | Operate 8.2.x Tasklist 8.2.x Identity 8.2.x | Optimize 3.10.x | 0.14.x  |
-| Web Modeler 8.2.x     | Zeebe 8.2.x | Operate 8.2.x Tasklist 8.2.x Identity 8.2.x | Optimize 3.10.x | 0.14.x  |
-| Web Modeler 8.3.x     | Zeebe 8.3.x | Operate 8.3.x Tasklist 8.3.x Identity 8.3.x | Optimize 8.3.x  | 1.3.x   |
-| Web Modeler 8.4.x     | Zeebe 8.4.x | Operate 8.4.x Tasklist 8.4.x Identity 8.4.x | Optimize 8.4.x  | 1.6.x   |
+This overview shows which Zeebe version works with which Modeler, Operate, Tasklist, and Optimize:
+
+| Design                | Automate    |                                             | Improve         |
+| --------------------- | ----------- | ------------------------------------------- | --------------- |
+| Desktop Modeler 4.12+ | Zeebe 1.3.x | Operate 1.3.x Tasklist 1.3.x IAM 1.3.x      | Optimize 3.7.x  |
+| Desktop Modeler 5.0+  | Zeebe 8.0.x | Operate 8.0.x Tasklist 8.0.x Identity 8.0.x | Optimize 3.8.x  |
+| Desktop Modeler 5.4+  | Zeebe 8.1.x | Operate 8.1.x Tasklist 8.1.x Identity 8.1.x | Optimize 3.9.x  |
+| Desktop Modeler 5.10+ | Zeebe 8.2.x | Operate 8.2.x Tasklist 8.2.x Identity 8.2.x | Optimize 3.10.x |
+| Web Modeler 8.2.x     | Zeebe 8.2.x | Operate 8.2.x Tasklist 8.2.x Identity 8.2.x | Optimize 3.10.x |
+| Web Modeler 8.3.x     | Zeebe 8.3.x | Operate 8.3.x Tasklist 8.3.x Identity 8.3.x | Optimize 8.3.x  |
+| Web Modeler 8.4.x     | Zeebe 8.4.x | Operate 8.4.x Tasklist 8.4.x Identity 8.4.x | Optimize 8.4.x  |
 
 :::note
 You can also use newer versions of Desktop and Web Modeler with older Zeebe versions.
 :::
 
+## form-js version matrix
```

Comment:
@akeller why is this needed from a customer perspective? Where is the source of this table?

Reply:
https://github.com/camunda/camunda-docs/pull/3300

This is the source. I merged it as a quick win, but we can move it elsewhere. I didn't have a better recommendation of where to put it.